prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.fhir

```diff
@@ -9675,6 +9693,36 @@
         "lastUpdated" : "2021-08-09T08:52:00-06:00"
       },
       "extension" : [ {
+        "url" : "https://reportstream.cdc.gov/fhir/StructureDefinition/filler-assigned-identifier",
+        "valueIdentifier" : {
+          "type" : {
+            "coding" : [ {
+              "system" : "http://terminology.hl7.org/CodeSystem/v2-0203",
+              "code" : "FILL",
+              "display" : "Filler Identifier"
+            } ]
+          },
+          "system" : "urn:id:21:AA:B0029251Sm12776123",
```

Comment:
The `"system" : "urn:id:21:AA:B0029251Sm12776123",  ` seems wrong to me in this case.  We may need to tweak the HL7->FHIR conversion

Reply:
So the problem with adding them to the identifier array is that there's no way to distinguish between this one and what was generated from ORC.3/OBR.3 (added below). That's what I was trying to express with [this comment](https://github.com/CDCgov/prime-reportstream/pull/6581/files/2090f964cab6c6490fc0c11e5c407c2db41de98f#r958571812).  I think this is also a part of the reason that originally we considered this to be an unnecessary mapping.
```identifier_2:
  condition: $valueIn NOT_NULL
  valueOf: datatype/Identifier_var
  generateList: true
  expressionType: resource
  vars:
    valueIn: ORC.3.1 | OBR.3.1
    systemCX: ORC.3.2 | OBR.3.2
    universalId: ORC.3.3 | OBR.3.3
    universalIdSystem: ORC.3.4 | OBR.3.4
  constants:
    system: "http://terminology.hl7.org/CodeSystem/v2-0203"
    code: "FILL"
    display: "Filler Identifier"
```

Reply:
Carlos and I talked offline
- OBR.2 and OBR.3 are the same as ORC.2 and ORC.3 and are supposed to go in the service request. We chose not to do that as it is a duplicate for the same information stored in diagnostic report. This makes sense to go in the extension instead of the identifier since this is not the identifier of the Service request in question it is the linked identifier to a different diagnostic report/service request.
- Will be leaving this extension as is.